### DISCUSSION TITLE
How do I select a column using an alias for output prisma orm?

### DISCUSSION THREAD

**Discussion by agustints** [prisma/prisma#8084] [Q&A] [answered]
How do I select a column using an alias for output prisma orm?

**Comment by ryands17** [comment, marked as answer]
@FluxusGaming 👋 

Currently this isn't supported. I would suggest opening a feature request for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
